om-2.0.rdf

```diff
@@ -31599,17 +31588,11 @@
 
   <owl:Class rdf:about="&om;Acidity">
     <rdfs:label xml:lang="en">acidity</rdfs:label>
+    <om:alternativeSymbol>pH level</om:alternativeSymbol>

```

Comment:
I believe this should be `om:alternativeLabel` - the symbol is still the same (i.e. "pH")!

Reply:
Oh yes. Sorry about that. I have modfied accordingly.